healsparse/healSparseMap.py

```diff
@@ -322,6 +322,26 @@ def getValuePixel(self, pixel, nest=True):
 
         return self._sparseMap[_pix + self._covIndexMap[ipnestCov]]
 
+    def HSPixel2RaDec(self, pixel):
+        """
+        Get the position ra, dec (in degrees) of the center of a certain HealSparse pixel
+        according to its covIndexMap
+        """
+
```

Comment:
Is pixel here the internal pixel index rather than a regular healpix pixel number?  Is this something that we need exposed?

Reply:
Hmmm actually, I think that there's no use case for this (I thought that it may have been useful at the time of generating the tests for `generateHealpixMap` but it is not the case). I'll remove them, then.